app/code/Magento/SalesSequence/Model/ResourceModel/Meta.php

```diff
@@ -92,6 +92,27 @@ public function loadByEntityTypeAndStore($entityType, $storeId)
         return $meta;
     }
 
+    /**
+     * Retrieves Metadata Ids by store id
+     *
+     * @param int $storeId
+     * @return int[]
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getIdsByStore($storeId)
```

Comment:
Introduction of a new public method is backwards incompatible and cannot be added to a patch release. Please create a new class for this functionality